### ISSUE TITLE
Sony RAW files incorrectly show up on Immich

### ISSUE THREAD

**Issue by ericswpark** [immich-app/immich#9845]
### The bug

Sony RAW images taken with my ZV-E1 show up incorrectly on Immich. Here is a side-by-side comparison with IrfanView:

![image](https://github.com/immich-app/immich/assets/13326074/94295439-6c45-4a62-905e-e7e02d880e57)

Left is Immich, right is IrfanView.

Attached RAW file to test with:

[DSC00021.zip](https://github.com/immich-app/immich/files/15479860/DSC00021.zip)

### The OS that Immich Server is running on

unRAID

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

N/A

### Platform with the issue

- [X] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
version: '3.8'

#
# WARNING: Make sure to use the docker-compose.yml of the current release:
#
# https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml
#
# The compose file on main may not be compatible with the latest release.
#

name: immich

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    command: ['start.sh', 'immich']
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    env_file:
      - .env
    ports:
      - 2283:3001
    depends_on:
      - redis
      - database
    restart: always

  immich-microservices:
    container_name: immich_microservices
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/hardware-transcoding
    #   file: hwaccel.transcoding.yml
    #   service: cpu # set to one of [nvenc, quicksync, rkmpp, vaapi, vaapi-wsl] for accelerated transcoding
    command: ['start.sh', 'microservices']
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    env_file:
      - .env
    depends_on:
      - redis
      - database
    restart: always

  immich-machine-learning:
    container_name: immich_machine_learning
    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
    # Example tag: ${IMMICH_VERSION:-release}-cuda
    image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/ml-hardware-acceleration
    #   file: hwaccel.ml.yml
    #   service: cpu # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
    volumes:
      - model-cache:/cache
    env_file:
      - .env
    restart: always

  redis:
    container_name: immich_redis
    image: registry.hub.docker.com/library/redis:6.2-alpine@sha256:51d6c56749a4243096327e3fb964a48ed92254357108449cb6e23999c37773c5
    restart: always

  database:
    container_name: immich_postgres
    image: registry.hub.docker.com/tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
    volumes:
      - ${DB_DATA_LOCATION}:/var/lib/postgresql/data
    restart: always

volumes:
  pgdata:
  model-cache:
```

### Your .env content

```Shell
# You can find documentation for all the supported env variables at https://immich.app/docs/install/environment-variables

# The location where your uploaded files are stored
UPLOAD_LOCATION=/mnt/user/immich

# The Immich version to use. You can pin this to a specific version like "v1.71.0"
IMMICH_VERSION=release

# Connection secret for postgres. You should change it to a random password
DB_PASSWORD=REDACTED

# The values below this line do not need to be changed
###################################################################################
DB_HOSTNAME=immich_postgres
DB_USERNAME=postgres
DB_DATABASE_NAME=immich
DB_DATA_LOCATION=/mnt/cache/appdata/immich/postgres

REDIS_HOSTNAME=immich_redis
```

### Reproduction steps

```bash
1. Upload any Sony RAW image (or use the example one provided above)
2. Observe bug
```

**Comment by mertalev** [comment]
The ZV-E1 isn't on libraw's [list of supported cameras](https://www.libraw.org/supported-cameras). I'm afraid there's not much for us to do here.

**Comment by mertalev** [comment]
Oh, it looks like they did add support for it, but it hasn't made it into a release yet. I'll keep this open as a reminder to update libraw when we can.

**Comment by RustedAperture** [comment]
> Oh, it looks like they did add support for it, but it hasn't made it into a release yet. I'll keep this open as a reminder to update libraw when we can.

Didn't want to open a new case since this seems to be related. 

Noticed that Fuji-film X-T5 has also been added to the public git but was not in the list of supported cameras. Similar behaviour where colours are not represented correctly on new Cameras. building libraw from the git rather than release might need to be looked at.

In the image below the left is Windows and the right is Immich

![X-T5](https://cloud.wfox.app/s/yxz9MoBRoExyQbd/preview)

**Comment by mertalev** [comment]
Hmm, libraw's release cadence is pretty sporadic - the next release is probably many months away. We could use the snapshot, but there have been a lot of bug fixes since then. Maybe we can just use main for now.

**Comment by mertalev** [comment]
In the meantime, you may have some luck with enabling the "extract embedded preview" option, re-running thumbnail generation and clearing your browser cache.

**Comment by RustedAperture** [comment]
> In the meantime, you may have some luck with enabling the "extract embedded preview" option, re-running thumbnail generation and clearing your browser cache.

Thank you, this helped part of it, Previews are good till the image is clicked now. Looking forward to the future :D

**Comment by mertalev** [comment]
Fixed via #10457
